### ISSUE TITLE
无法复现实验结果

### ISSUE THREAD

**Issue by xpz123** [zzw922cn/Automatic_Speech_Recognition#22]
你好，我现在用这个实验的代码在timit进行了实验。但是在test集上得到的错误率大概在0.35左右，与主页上的图标的实验结果相差很多。而且，主页上图标的训练集和测试集的错误率下降曲线的横坐标不是很明确。不太了解是epoch还是什么呢？我目前选用的参数是2层blstm，learning rate 0.0001，也就是脚本中默认的模型参数。所以想问一下，如何能够复现出如图中所示的实验结果？是否有其他的trick或是用了不同的模型参数呢？
多谢

**Comment by zzw922cn** [comment]
@xpz123 你好！曲线中的横坐标是Epoch的，并且，我在训练过程中learning rate有过多次调整，从0.001调整到0.00001，另外，训练中使用了dropout以及batch normalization（github代码中可能没有加上BN）等技巧。

**Comment by xpz123** [comment]
Hi, 多谢你的回复~
但是，那两个图的横坐标如果都是epoch的话，我还有一点不太清楚的地方。就是这个train的图和test的应该不是反映的一次实验结果吧？以为test集合上可以观察到在20epoch不到的地方网络已经收敛到一个不错的结果。但是train的图中，在50epoch的地方还没有收敛。或者说test的图中的横坐标是要乘以10？

**Comment by zzw922cn** [comment]
@xpz123 你好，那是因为我不是每一个Epoch都进行test，而是隔几个Epoch再进行一次test。

**Comment by lz1313** [comment]
I can reproduce the test PER as about 24%~25%, is this the best result your code can achieve?
Can you share all the hyperparameters like lr, activation, grad_clip, decay_steps, decay_rate, etc. you are using to achieve the PER in your plot?
Thanks

**Comment by YJMA3** [comment]
你好，我想问一下你主页里面test的结果很接近20%，能不能说一下你在训练的时候用的是什么参数？能把所有的参数说一下吗？包括模型，epoches，BN，learning rate，dropout等所有的参数。
